# motor sensor data message from DrRobot Robot.

Header header    	# timestamp in the header is the time the driver
		 	# returned the battery/power reading
uint32 motorID		# motor ID number
int32 encoder_pos	# encoder positon count
int32 encoder_vel	# encoder velocity value
float32 motor_temperature # motor temperature sensor reading
int32 motor_pwm	# output PWM value, only for Jaguar series robot
